### ISSUE TITLE
[Bug][deployment] bind 80 failed for config-ui container with containerd

### ISSUE THREAD

**Issue by matrixji** [apache/incubator-devlake#2290]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.

### What happened

Deploy with the master branch (using helm), Kubernetes 1.24 using contained, and seems contained does not support bind with 80/443 by default settings. 
So prefer to use 4000 port for config-ui container. This will keep the same settings as dev, and it will not require privileged users for bind the port.

ref: https://github.com/containerd/containerd/issues/2516

Error for config-ui container:
```plain
[root@nuc ~]# kubectl logs  devlake0-ui-67ccb4cf4b-6wwk7
2022/06/21 17:46:06 [emerg] 12#12: bind() to 0.0.0.0:80 failed (13: Permission denied)
nginx: [emerg] bind() to 0.0.0.0:80 failed (13: Permission denied)
[root@nuc ~]#
```

### What you expected to happen

Hope nginx could run with non-root user in any container environment.

### How to reproduce

helm install or kubectl apply yaml under Kubernetes with contained, and check the config-ui's pod's log.

### Anything else

N/A

### Version

main

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by Startrekzky** [comment]
Thanks for reporting the issue. I'm looking forward to your PR.

**Comment by klesh** [comment]
Fixed by #2375, will be released v0.12.
